### ISSUE TITLE
Use more delicate view resize steps

### ISSUE THREAD

**Issue by tobealive** [microsoft/vscode#149488]
The resize steps when using the `increaseViewWidth` / `..Height` commands / shortcuts  are pretty heavy. 
Being a keyboard driven navigator through the editor I would  love to have an option to determine the step size for those commands, because I think this way no users would need to compromise.

If it's needed, I would put my bit of work into that if you would like to have such an option. 
Even if there shall not be an option, it would be marvelous of you to reduce the current default step size by just 1 percent. This  keeps it still fairly big if someone prefers those step sizes but already allows more precise resizing with the keyboard so the view / splits can fit one's needs.

Basically:
`protected static readonly RESIZE_INCREMENT = 6.5; // This is a media-size percentage`
to
`protected static readonly RESIZE_INCREMENT = 5.5; // This is a media-size percentage`

Imho even smaller step sizes would be better

**Comment by sbatten** [comment]
seems like a fair request

**Comment by sbatten** [comment]
After thinking about this, I decided to move away from using view percentage increments. Instead, we now use css px units so that the resize increments are consistent. The newly chosen increment size is ~equivalent to 5.5% of a 1080px width monitor.

**Comment by tobealive** [comment]
Great to heat. Thank you!

**Comment by ash211** [comment]
I run vscode on a 27" 1440p monitor in vertical orientation, and after a recent vs code upgrade now my `workbench.action.terminal.resizePaneUp` command is increasing the terminal height by only 1 line, instead of ~10 lines like before.  Is that expected from this change?

The size increase now feels very small, and I'm retraining my fingers to expand the terminal directly to full-screen (Ctrl+Shift+Q) instead of expanding the terminal 4 or 5 times (Ctrl+Cmd+Up).  Now, to use (Ctrl+Cmd+Up) I'd have to press ~40 to ~50 times (or hold) to get it to a mostly-expanded size.

**Comment by tobealive** [comment]
To meet your needs with the current implementation, you could map your old key to a multi-command or macro. Using the corresponding vscode extension [multi-command](https://marketplace.visualstudio.com/items?itemName=ryuta46.multi-command) / [macros](https://marketplace.visualstudio.com/items?itemName=geddski.macros).That way you could control the exact resize step amount.

> I'm retraining my fingers to expand the terminal directly to full-screen (Ctrl+Shift+Q)

that make it sounds like a tough thing to do 🏋️‍. As far as I could tell until now a human brain should be able to do it in a few minutes or a handful of repetitions 🤔.
